### PR TITLE
Add kt_android_local_test demo

### PR DESCRIPTION
**Pull request by utzcoz**
Fix https://github.com/robolectric/robolectric-bazel/issues/8.

### CODE REVIEW

**Comment by utzcoz** [comment]
Hi @Bencodes @jin @cheister , could you help to review this PR? Thanks.

**Comment by utzcoz** [comment]
I don't know why tasks I added in `.bazelci/examples.yml` for `kt_android_local_test` didn't run.

**Comment by utzcoz** [comment]
> Hi @Bencodes @jin @cheister , could you help to review this PR? Thanks.

Friendly ping, @Bencodes @jin @cheister could you help to review this PR? Thanks.

**Comment by Bencodes** [comment]
@utzcoz I don't have merge access but @shs96c might be able to help.

File: examples/kt_android_local_test/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+To run the Robolectric test in this workspace:
+
+```
+$ bazel test //src/test:main_activity_test
+```
+
+Roboelectric tests in Bazel requires special setup to generate a file called
+`robolectric-deps.properties`, containing information about android-all jars and
+their locations. You can find this setup in the
+[bazel/robolectric](bazel/roboelectric/) directory.
```
Comment (jin) [inline examples/kt_android_local_test/README.md:12]: Where is this?

Comment (utzcoz) [reply examples/kt_android_local_test/README.md:12]: The README.md is copied from android_local_test module. Looks like `robolectric-deps.properties` doesn't exist in sample. What about sending a new PR to remove them from those local test modules? I can do it at this week. If you prefer to it is removed at this PR, I can update new commit for it later this week.

Comment (jin) [reply examples/kt_android_local_test/README.md:12]: I think it's referring to the bazel directory in the upstream robolectric repo: https://github.com/robolectric/robolectric-bazel/blob/6d18a0ea372991da97e46b6d541dff3dfacb61b7/bazel/BUILD#L22

Can you update the text here and in the android_local_test directory to reflect that?

Comment (utzcoz) [reply examples/kt_android_local_test/README.md:12]: Sure. I will update it this weekend.

Comment (utzcoz) [reply examples/kt_android_local_test/README.md]: @jin I updated link of `robolectric-deps.properties` of `README.md` with new commit.

**Comment by utzcoz** [comment]
Hi @jin I think it can be review again.

**Review by jin — APPROVED** [review]

**Pull request merged**